components/brave_wallet/browser/eth_tx_controller.cc

```diff
@@ -253,6 +253,23 @@ void EthTxController::ContinueAddUnapprovedTransaction(
   uint256_t gas_limit;
   if (!success || !HexValueToUint256(result, &gas_limit)) {
     gas_limit = 0;
+    mojom::TransactionType tx_type;
+    if (GetTransactionInfoFromData(ToHex(tx->data()), &tx_type, nullptr,
+                                   nullptr)) {
+      // Try to use reasonable values when we can't get an estimation.
+      // These are taken via looking through the different types of transactions
+      // on etherscan and taking the next rounded up value for the largest found
+      if (tx_type == mojom::TransactionType::ETHSend) {
```

Comment:
We won't try to fetch gas limit for this type in the first place actually. But not harmful having this here.

Reply:
Ya I realized when added but thought why not and might protect against some future change